precompiles/randomness/src/lib.rs

```diff
@@ -57,9 +59,17 @@ pub enum Action {
 
 // Tests to verify equal to weight_to_gas(weight) in runtime integration tests
 pub const REQUEST_RANDOMNESS_ESTIMATED_COST: u64 = 26325;
-pub const FULFILLMENT_OVERHEAD_ESTIMATED_COST: u64 = 24545;
 pub const INCREASE_REQUEST_FEE_ESTIMATED_COST: u64 = 16718;
 pub const EXECUTE_EXPIRATION_ESTIMATED_COST: u64 = 21989;
+
+/// Fulfillment overhead cost, which takes input weight hint -> weight -> return gas
+pub fn fulfillment_overhead_cost<T: pallet_evm::Config>(num_words: u8) -> u64 {
```

Comment:
```suggestion
pub fn fulfillment_overhead_cost<T: pallet_evm::Config>(num_words: u8) -> Weight {
```

This would help clear up ambiguity over whether this is weight or gas

Reply:
err, it's gas, not weight, isn't it...?